### ISSUE TITLE
row.set edited the whole row instead only that cell

### ISSUE THREAD

**Issue by quarkhadron** [theoephraim/node-google-spreadsheet#670]
I used the Google Sheets QUERY function to visualize data from another sheet but when I use row.set instead of updating only that cell, it replaced the whole row which make QUERY function useless and the code broken

**Comment by theoephraim** [comment]
yeah unfortunately this is a limit of using the [values based api methods](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values) which the row-based calls depend on.

Would take some thought to figure out how to better support more complex cases while still maintaining the easy DX of the row-based methods...

In the meantime, I'd suggest using the cell-based methods
